policy/modules/apps/ipmitool.if

```diff
@@ -0,0 +1,88 @@
+## <summary>Intelligent Platform Management Interface (IPMI) Tool</summary>
+
+########################################
+## <summary>
+##	Execute the ipmitool program in
+##	the ipmitool domain.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed to transition.
+##	</summary>
+## </param>
+#
+interface(`ipmitool_domtrans',`
+	gen_require(`
+		type ipmitool_t, ipmitool_exec_t;
+	')
+
+	corecmd_search_bin($1)
+	domtrans_pattern($1, ipmitool_exec_t, ipmitool_t)
+
+	allow $1 ipmitool_t:process2 nnp_transition;
```

Comment:
Need to drop this now that there is a separate nnp interface.

Reply:
Yup, sorry, not sure how I missed that.